enclave_wrangler/objects_api.py

```diff
@@ -519,8 +520,8 @@ def fetch_cset_and_member_objects(
         return
 
     # Expression items & concept set members
-    expression_items = []
-    member_items = []
+    expression_items = set()
+    member_items = set()
```

Comment:
## Bug fix: `UniqueViolation: duplicate key value violates unique constraint "concept_pkey1"`
I got the error at first and then couldn't replicate. I think I know what happened.

Previously we set it up to add any concepts to `concept` table. However, there haven't been any new concepts in a long time until the vocab update that just happened.

Now that there are new concepts, it is trying to add to the DB. There _is_ a check where it first sees if the row is in the DB. If it is, it skips adding. That check was working correctly.

So why did this happen?
I think it's because there were _multiple_ instances of the new concept being used. So when it checked, the concept wasn't in the DB. Then, it added the concept once (as it was a member of 1 csets). However, it _kept trying to add_ the same concept again because it was a member of other csets.

How I fixed
I believe this is fixed now by changing it so that when concept members are being identified, it adds them to a set, rather than a list. This should eliminate duplicates, so it will now only try to insert once.